test/org.test.ts

```diff
@@ -0,0 +1,129 @@
+import {ethers} from "@nomiclabs/buidler";
+import {assert, expect} from "chai";
+import {submit} from "./support";
+import {OrgFactory} from "../contract-bindings/ethers";
+
+describe("Org", function () {
+  it("should allow a project to be anchored", async function () {
+    const [owner] = await ethers.getSigners();
+    const ownerAddr = await owner.getAddress();
+
+    const org = await new OrgFactory(owner).deploy(ownerAddr);
+
+    await org.deployed();
+
+    const id = ethers.utils.randomBytes(32);
+    const rev = ethers.utils.randomBytes(32);
+    const hash = ethers.utils.randomBytes(32);
+
+    // Create a new anchor.
+
+    assert.equal(await org.projectExists(id), false);
+
+    await submit(org.connect(owner).anchorProject(id, rev, hash));
+
+    assert.equal(await org.projectExists(id), true);
+
+    let proj = await org.projects(id);
+
+    assert.equal(proj.rev, ethers.utils.hexlify(rev));
+    assert.equal(proj.hash, ethers.utils.hexlify(hash));
+
+    // Update the anchor.
+
+    const newHash = ethers.utils.randomBytes(32);
+    await submit(org.connect(owner).anchorProject(id, rev, newHash));
+
+    proj = await org.projects(id);
+
+    assert.equal(proj.hash, ethers.utils.hexlify(newHash));
+
+    // Remove the anchor.
+
+    await submit(org.connect(owner).removeProject(id));
+    assert.equal(await org.projectExists(id), false);
+  });
+
+  it("should allow removing a project even if it doesn't exist", async function () {
+    const [owner] = await ethers.getSigners();
+    const ownerAddr = await owner.getAddress();
+
+    const org = await new OrgFactory(owner).deploy(ownerAddr);
+
+    const id = ethers.utils.randomBytes(32);
+    await submit(org.connect(owner).removeProject(id));
+  });
+
+  it("should only allow the org owner to anchor", async function () {
+    const [owner, bob] = await ethers.getSigners();
+    const ownerAddr = await owner.getAddress();
+
+    const org = await new OrgFactory(owner).deploy(ownerAddr);
+
+    await org.deployed();
+
+    const id = ethers.utils.randomBytes(32);
+    const rev = ethers.utils.randomBytes(32);
+    const hash = ethers.utils.randomBytes(32);
+
+    // Should revert!
+    await submit(org.connect(bob).anchorProject(id, rev, hash))
+      .then(() => expect.fail("Expected error"))
+      .catch(() => {});
```

Comment:
Not sure this is the best way to expect failure, but couldn't get other ways to work.